src/main.rs

```diff
@@ -231,17 +237,20 @@ async fn push(args: &ArgMatches) -> anyhow::Result<()> {
     })?);
 
     // Hippo configuration
-    let hippo_url = args.value_of(ARG_HIPPO_URL).map(|s| s.to_owned());
+    let hippo_url = args
+        .value_of(ARG_HIPPO_URL)
+        .map(|s| s.to_owned())
+        .ok_or_else(|| anyhow::anyhow!("A Hippo url is requred. Use --hippo-url or $HIPPO_URL"));
```

Comment:
Capitalise URL though the `required` option should make sure this never happens I think! Typo in "required" too.
 
```suggestion
        .ok_or_else(|| anyhow::anyhow!("A Hippo URL is required. Use --hippo-url or $HIPPO_URL"));
```

Reply:
Does this need a `?`, or is it intentional that it continues for now and fails when you construct the notifier settings?

Reply:
It was intentional, but upon review, I think I can reduce a little complexity later if I `?` out here.